imports/client/components/PuzzlePage.tsx

```diff
@@ -545,37 +680,36 @@ const PuzzlePageMetadata = ({
   const numGuesses = guesses.length;
 
   const answersElement = correctGuesses.length > 0 ? (
-    <span className="puzzle-metadata-answers">
+    <PuzzleMetadataAnswers>
       {
         correctGuesses.map((guess) => (
-          <span key={`answer-${guess._id}`} className="answer tag-like">
+          <PuzzleMetadataAnswer key={`answer-${guess._id}`}>
             <span>{guess.guess}</span>
             {!hasGuessQueue && (
-              <Button className="answer-remove-button" variant="success" onClick={() => onRemoveAnswer(guess._id)}>&#10006;</Button>
+              <AnswerRemoveButton variant="success" onClick={() => onRemoveAnswer(guess._id)}>
```

Comment:
Not in scope, but this should really be a `useCallback`, right?

Reply:
Yeah, probably the right answer is to have an actual function component for each guess, instead of defining them inline, and then yes, that component would call `useCallback`.